src/Modal/Modal.jsx

```diff
@@ -29,27 +30,33 @@ export class Modal extends React.Component {
   }
 
   handleResize() {
-    this.setState({windowHeight: window.innerHeight});
+    // The ensures the modal can scroll when it exceeds 90% of the window height
+    const scrollable = this.modal && this.modal.scrollHeight >= (0.9 * window.innerHeight) - 60;
+    this.setState({
+      windowHeight: window.innerHeight,
+      scrollable,
+    });
   }
 
   render() {
     let windowStyle = {
       width: `${this.props.width}px`,
       marginLeft: `-${this.props.width / 2}px`,
     };
-    // The content is max 90% of the window height less 70px (height of the header)
+    // The content is max 90% of the window height less 60px (height of the header)
     let contentStyle = {
-      maxHeight: (this.state.windowHeight * 0.9) - 70,
+      maxHeight: (this.state.windowHeight * 0.9) - 60,
+      overflowY: this.state.scrollable ? "scroll" : "hidden",
     };
     return (
       <div className="Modal">
         <div className="Modal--background" onClick={this.props.closeModal} />
-        <div className="Modal--window" style={windowStyle} >
-          <div className="Modal--header">
+        <div className="Modal--window" style={windowStyle}>
+          <header className="Modal--header">
             <button className="Modal--close" onClick={this.props.closeModal} />
             <h2>{this.props.title}</h2>
-          </div>
-          <div style={contentStyle} className="Modal--window--content">
+          </header>
+          <div style={contentStyle} className="Modal--window--content" ref={(ref) => {this.modal = ref;}}>
```

Comment:
using it to compare the modal's scrollHeight to the window's height, so we can enable scrolling when it's needed. this is the best way I could think of to fix the issue where the scrollbar always showed up